be/Service/UsersSvc.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace Service;
+
+use Doctrine\ORM\EntityManager;
+use Doctrine\Common\Collections\Collection;
+use Model\Entity\Game;
+use Model\Entity\User;
+use Model\Repository\UserRepository;
+use Model\Factory\GameFactory;
+use Service\RatingCalculator;
+use Slim\Http\Response;
+use \Psr\Http\Message\ServerRequestInterface as Request;
+
+class UsersSvc
```

Comment:
Ta klasa ma 3 metody publiczne które robią co innego i zastanawiam się czy tak naprawdę to jeszcze nie powinno być rozbite na 3 oddzielne klasy:
- UsersGetSvc
- UserAddSvc
- UserUpdateRatings

Czy jakieś takie. Co o tym sądzisz ?